tests/py/casmutils/xtal/coordinate.py.in

```diff
@@ -0,0 +1,97 @@
+#!@PYTHON@
```

Comment:
I don't know if this is the case, but if you use `git mv` instead of just renaming files, I think it'll avoid considering the file renaming a giant change.

Reply:
i did '''git mv''' only. But since I added a new test for apply sym op, I guess it is considering as a big change. Sorry about that.